README.md

```diff
@@ -26,339 +26,213 @@
+  client options
+- Send input data (e.g. a jpeg image) directly to Triton in the [body of an HTTP 
+  request without any additional metadata](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_binary_data.md#raw-binary-request)
+
+### Extend Triton
+
+[Triton Inference Server's archicture](docs/architecture.md) is specifically 
+designed for modularity and flexibility
+
+- [Customize Triton Inference Server](docs/compose.md) for your use case
+- [Create custom backends](https://github.com/triton-inference-server/backend)
+  in either [C/C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api)
+  or [Python](https://github.com/triton-inference-server/python_backend)
+- Create [decouple backend and models](docs/decoupled_models.md) can send 
+  multiple responses for a request or not send any responses for a request
+- Use a [Triton repository agent](docs/repository_agents.md) to add functionality
+  that operates when a model is loaded and unloaded, such as authentication, 
+  decryption, or conversion
+- Deploy Tirton on [Jetson and JetPack](jetson.md)
+
+### Additional Documentation
+
+- [FAQ](docs/faq.md)
+- [User Guide](docs#user-guide)
+- [Developer Guide](docs#developer-guide)
+- [Release Notes](https://docs.nvidia.com/deeplearning/triton-inference-server/release-notes/index.html)
+- [GPU, Driver, and CUDA Support
+Matrix](https://docs.nvidia.com/deeplearning/dgx/support-matrix/index.html)
 
 ## Contributing
 
 Contributions to Triton Inference Server are more than welcome. To
-contribute make a pull request and follow the guidelines outlined in
-[CONTRIBUTING.md](CONTRIBUTING.md). If you have a backend, client,
+contribute make please review the [contribution 
+guidelines](CONTRIBUTING.md). If you have a backend, client,
 example or similar contribution that is not modifying the core of
 Triton, then you should file a PR in the [contrib
 repo](https://github.com/triton-inference-server/contrib).
 
 ## Reporting problems, asking questions
 
-We appreciate any feedback, questions or bug reporting regarding this
-project. When help with code is needed, follow the process outlined in
-the Stack Overflow (<https://stackoverflow.com/help/mcve>)
-document. Ensure posted examples are:
-
-* minimal – use as little code as possible that still produces the
+We appreciate any feedback, questions or bug reporting regarding this project. 
+When posting [issues in GitHub](https://github.com/triton-inference-server/server/issues),
+follow the process outlined in the [Stack Overflow document](https://stackoverflow.com/help/mcve).
+Ensure posted examples are:
+- minimal – use as little code as possible that still produces the
   same problem
-
-* complete – provide all parts needed to reproduce the problem. Check
+- complete – provide all parts needed to reproduce the problem. Check
   if you can strip external dependency and still show the problem. The
```

Comment:
~dependency~ dependencies

Reply:
Funny, this one is in the current README :-)